pkg/clients/handlers.go

```diff
@@ -29,12 +32,27 @@ type PaginationLinksResponse struct {
 	Next  string `json:"next,omitempty"`
 }
 
-func (a *API) RegisterEndpoints(mux *chi.Mux) {
+func (a *API) RegisterEndpoints(mux *chi.Mux) *API {
```

Comment:
```suggestion
func (a *API) RegisterEndpoints(mux *chi.Mux) {
```

as discussed, we don't really use the returning object
chaining pattern tends to be a pain to test, so if we can avoid it, better